layouts/partials/docs/api-reference-links.html

```diff
@@ -28,6 +28,8 @@
     <!-- Loop through sections -->
     {{- range $apiRefSection := $apiRefSections.Sections -}}
         {{- $apiRefDir := $apiRefSection.RelPermalink -}}
+        {{- $fragmentApiRefDir := split $apiRefDir "/docs" -}}
+        {{- $apiRefDir := printf "/docs%s" (index ($fragmentApiRefDir) 1) -}}
```

Comment:
Even better: don't repeat the string `/docs`

Either calculate it from [`.Page.Sections`](https://gohugo.io/methods/page/sections/) (best), or hard code it once using a variable to repeat it.